kustomize/base/brigadasos-db.yaml

```diff
@@ -24,12 +27,45 @@ metadata:
     app: brigadasos-db
 spec:
   storageClassName: manual
+  volumeName: brigadasos-db-pvolume
   accessModes:
     - ReadWriteMany
   resources:
     requests:
       storage: 50Gi
 ---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: brigadasos-db-temp-pvolume
+  namespace: nadedb
+  labels:
+    type: local
+    app: brigadasos-db
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 50Gi
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/mnt/nadedb"
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: brigadasos-db-temp-pvolume-claim
+  namespace: nadedb
+  labels:
+    app: brigadasos-db
+spec:
+  storageClassName: manual
+  accessModes:
```

Comment:
This new `PersistentVolumeClaim` should also have a `volumeName` reference for mapping, same as what you modified above. Something like `volumeName: brigadasos-db-temp-pvolume`